drivers/pwm/pwm_nrfx.c

```diff
@@ -198,7 +334,7 @@ static int pwm_nrfx_pm_control(struct device *dev,
 
 #define PWM_NRFX_OUTPUT_PIN(dev_idx, ch_idx)		      \
 	(DT_NORDIC_NRF_PWM_PWM_##dev_idx##_CH##ch_idx##_PIN | \
-	 (DT_NORDIC_NRF_PWM_PWM_##dev_idx##_CH##ch_idx##_INVERTED ?   \
+	(DT_NORDIC_NRF_PWM_PWM_##dev_idx##_CH##ch_idx##_INVERTED ?   \
```

Comment:
This was okay.  The second line should be indented with one space because its opening parenthesis is the inner one. 
What could be corrected here instead is the alignment (actually, lack of it) of the `\` characters.

Reply:
(Y)